libs/blocks/media/media.js

```diff
@@ -30,9 +30,10 @@ function decorateAvatar(el) {
 function decorateQr(el) {
   const text = el.querySelector('.text');
   if (!text) return;
-  const appStore = text.children[(text.children.length - 1)];
-  const googlePlay = text.children[(text.children.length - 2)];
   const qrImage = text.children[(text.children.length - 3)];
+  const qrCodeLinks = el.querySelectorAll('a');
+  const appStore = qrCodeLinks[1];
+  const googlePlay = qrCodeLinks[0];
```

Comment:
Should we add a fail check?
```suggestion
  const qrCodeLinks = el.querySelectorAll('a');
  const appStore = qrCodeLinks[1];
  const googlePlay = qrCodeLinks[0];
  if(!appStore || !googlePlay) return;
```

Reply:
Added